Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Rundll32.exe_executes_a_rare_unsigned_module.yml

```diff
@@ -0,0 +1,1531 @@
+        3. The DLL Reputation is Suspicious.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "9"
+      Remediation:
+      - "39"
+    separatecontext: false
+    conditions:
+    - label: Remediation
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: Core.RiskyHost.risk_level
+            iscontext: true
+          right:
+            value:
+              simple: med
+          ignorecase: true
+      - - operator: notIn
+          left:
+            value:
+              simple: dll_extension
+            iscontext: true
+          right:
+            value:
+              simple: cpl,exe,pkg,ax,tmp,dll
+        - operator: isEqualNumber
+          left:
+            value:
+              complex:
+                root: Indicators
+                filters:
+                - - operator: containsGeneral
+                    left:
+                      value:
+                        simple: Indicators.Value
+                      iscontext: true
+                    right:
+                      value:
+                        simple: Core.OriginalAlert.event.action_module_sha256
+                      iscontext: true
+                    ignorecase: true
+                accessor: Score
+            iscontext: true
+          right:
+            value:
+              simple: "2"
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 803,
+          "y": 445
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "39":
+    id: "39"
+    taskid: c904e9d8-1fde-4e70-8259-0484f9c4ae96
+    type: title
+    task:
+      id: c904e9d8-1fde-4e70-8259-0484f9c4ae96
+      version: -1
+      name: Related Alerts
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "11"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1194
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "40":
+    id: "40"
+    taskid: 2c04ee78-190a-422d-a510-76030ba904c0
+    type: regular
+    task:
+      id: 2c04ee78-190a-422d-a510-76030ba904c0
+      version: -1
+      name: Close Alert - False Positive
+      description: commands.local.cmd.close.inv
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "31"
+    scriptarguments:
+      closeNotes:
+        simple: Resolved as False positive - Handled by the playbook "Rundll32.exe executes a rare unsigned module"
```

Comment:
Resolved as false positive - handled by the playbook "Rundll32.exe executes a rare unsigned module".

Reply:
Done